### ISSUE TITLE
Unable to install SharpZipLib from NuGet in my UWP application

### ISSUE THREAD

**Issue by zipgenius** [icsharpcode/SharpZipLib#620]
### Steps to reproduce
1. Create an UWP new app
2. Search NuGet package
3. Trying to install it

### Expected behavior
Package is installed and ready to use.

### Actual behavior
Package setup fails stating that it doesn't find any compatible platform. If I clone the repo and reference it into my UWP app, it wokrs like a charm.

### Version of SharpZipLib
1.3.1

### Obtained from (only keep the relevant lines)
- Package installed using NuGet

**Comment by christophwille** [comment]
That is strange because https://devblogs.microsoft.com/dotnet/announcing-uwp-support-for-net-standard-2-0/ is available for ages.

**Comment by zipgenius** [comment]
> That is strange because https://devblogs.microsoft.com/dotnet/announcing-uwp-support-for-net-standard-2-0/ is available for ages.

Yeah, I know that. It's really strange but I've tried with two different PCs and result is always the same.

**Comment by christophwille** [comment]
I just tried with what I had on my machine:

![image](https://user-images.githubusercontent.com/344208/117402138-20ce7d00-af06-11eb-8eb4-f0c6c972f046.png)

And this project allowed me to install SharpZipLib. You ain't by chance targeting a too-low min version?

**Comment by zipgenius** [comment]
Nothing to do. Here is what I obtain with a new blank UWP app, Build 19041 as target, Build 17763 as minimum.

It's in italian, so it states that the package "is not compatible with uap10.0.17763 (UAP.Version=v.10.0.17763). The package SharpZipLib 1.3.1 doesn't support any destination framework".

PM> install-package sharpziplib
Ripristino dei pacchetti per C:\Users\matte\source\repos\App19\App19.csproj in corso...
install-package : NU1202: Il pacchetto SharpZipLib 1.3.1 non è compatibile con uap10.0.17763 (UAP,Version=v10.0.17763). Il pacchetto SharpZipLib 1.3.1 non supporta alcun 
framework di destinazione.
In riga:1 car:1
+ install-package sharpziplib
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
install-package : NU1202: Il pacchetto SharpZipLib 1.3.1 non è compatibile con uap10.0.17763 (UAP,Version=v10.0.17763)/win10-arm. Il pacchetto SharpZipLib 1.3.1 non supporta 
alcun framework di destinazione.
In riga:1 car:1
+ install-package sharpziplib
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
install-package : NU1202: Il pacchetto SharpZipLib 1.3.1 non è compatibile con uap10.0.17763 (UAP,Version=v10.0.17763)/win10-arm-aot. Il pacchetto SharpZipLib 1.3.1 non supporta 
alcun framework di destinazione.
In riga:1 car:1
+ install-package sharpziplib
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
install-package : NU1202: Il pacchetto SharpZipLib 1.3.1 non è compatibile con uap10.0.17763 (UAP,Version=v10.0.17763)/win10-arm64-aot. Il pacchetto SharpZipLib 1.3.1 non 
supporta alcun framework di destinazione.
In riga:1 car:1
+ install-package sharpziplib
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
install-package : NU1202: Il pacchetto SharpZipLib 1.3.1 non è compatibile con uap10.0.17763 (UAP,Version=v10.0.17763)/win10-x64. Il pacchetto SharpZipLib 1.3.1 non supporta 
alcun framework di destinazione.
In riga:1 car:1
+ install-package sharpziplib
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
install-package : NU1202: Il pacchetto SharpZipLib 1.3.1 non è compatibile con uap10.0.17763 (UAP,Version=v10.0.17763)/win10-x64-aot. Il pacchetto SharpZipLib 1.3.1 non supporta 
alcun framework di destinazione.
In riga:1 car:1
+ install-package sharpziplib
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
install-package : NU1202: Il pacchetto SharpZipLib 1.3.1 non è compatibile con uap10.0.17763 (UAP,Version=v10.0.17763)/win10-x86. Il pacchetto SharpZipLib 1.3.1 non supporta 
alcun framework di destinazione.
In riga:1 car:1
+ install-package sharpziplib
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
install-package : NU1202: Il pacchetto SharpZipLib 1.3.1 non è compatibile con uap10.0.17763 (UAP,Version=v10.0.17763)/win10-x86-aot. Il pacchetto SharpZipLib 1.3.1 non supporta 
alcun framework di destinazione.
In riga:1 car:1
+ install-package sharpziplib
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
install-package : Ripristino dei pacchetti non riuscito. Verrà eseguito il rollback delle modifiche per 'App19'.
In riga:1 car:1
+ install-package sharpziplib
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
Tempo trascorso: 00:00:04.2363781

**Comment by zipgenius** [comment]
I investigated further and found that package manager is right: I don't have any compatible platform installed but just the following:

PS C:\Users\matte> dotnet --list-sdks
3.1.408 [C:\Program Files\dotnet\sdk]
5.0.101 [C:\Program Files\dotnet\sdk]
5.0.104 [C:\Program Files\dotnet\sdk]
5.0.202 [C:\Program Files\dotnet\sdk]

Actually, SharpZipLib works really well when referenced from the local repo, so it is compatible with those platforms. Maybe you might just add these platforms to the NuGet Package script.

**Comment by piksel** [comment]
The dotnet core sdks are not the same as frameworks. We currently target .NET framework 4.5 and .NET Standard 2.0, you can look at the compability matrix here:
https://docs.microsoft.com/en-us/dotnet/standard/net-standard

I don't know what is different on your computer, but hopefully it will work correctly after the next release, where #609 should be resolved.

**Comment by zipgenius** [comment]
UPDATE!
I tried to install 1.3.2 and... it installs!
![image](https://user-images.githubusercontent.com/547604/117637451-19b3a300-b182-11eb-8ed0-7ee63692add6.png)

I don't know whats happened, though.
